pkg/spec/spec.go

```diff
@@ -487,21 +488,29 @@ func (config *CreateConfig) cgroupDisabled() bool {
 	return config.Cgroup.Cgroups == "disabled"
 }
 
-func BlockAccessToKernelFilesystems(privileged, pidModeIsHost bool, g *generate.Generator) {
+func BlockAccessToKernelFilesystems(privileged, pidModeIsHost bool, mask, unmask string, g *generate.Generator) {
+	defaultMaskPaths := []string{"/proc/acpi",
```

Comment:
Can we move that outside the func on the package level? That would spare us from allocating the memory for each call.

Reply:
@baude, do we still need to care about `pkg/spec`?

Reply:
No, we definitely do not. It's gone in a few days.

Reply:
Okay, will get rid of the changes here, thanks!

Reply:
Fixed